apps/sap-commerce-cloud/package.json

```diff
@@ -6,11 +6,12 @@
   "dependencies": {
     "@contentful/app-scripts": "^1.7.13",
     "@contentful/app-sdk": "^4.23.0",
+    "@contentful/f36-button": "^4.68.1",
```

Comment:
Curious if it makes sense to remove this dependency and just use the button from the `@contentful/f36-components` package?

Reply:
I saw an example in the migration guide for the Icon Button that imported it from there so I just didn't realize I could get it from the main component library as well. I agree that makes sense then and I'll make that change.